services/QuillLMS/client/app/bundles/Staff/helpers/evidence/miscHelpers.ts

```diff
@@ -161,6 +171,72 @@ export const validateForm = (keys: string[], state: any[], ruleType?: string) =>
   return errors;
 }
 
+export function validateFormSection({
+  label,
+  activityPassages,
+  activityBecausePrompt,
+  activityButPrompt,
+  activitySoPrompt,
+}) {
+  switch(label) {
+    case titleCase(TEXT):
+      const passagePresent = activityPassages && activityPassages[0] && activityPassages[0].text && activityPassages[0].text !== BREAK_TAG;
+      return getCheckIcon(passagePresent);
+    case BUILDING_ESSENTIAL_KNOWLEDGE:
+      const essentialKnowledgePresent = (
+        activityPassages && activityPassages[0] &&
+        !!activityPassages[0].essential_knowledge_text &&
+        activityPassages[0].essential_knowledge_text !== ESSENTIAL_KNOWLEDGE_TEXT_FILLER &&
+        activityPassages[0].essential_knowledge_text !== BREAK_TAG
+      );
+      return getCheckIcon(essentialKnowledgePresent);
+    case HIGHLIGHTING_PROMPT:
+      const highlightingPresent = (activityPassages && activityPassages[0] && !!activityPassages[0].highlight_prompt && activityPassages[0].highlight_prompt !== DEFAULT_HIGHLIGHT_PROMPT);
+      return getCheckIcon(highlightingPresent);
+    case IMAGE:
+      const imageDetailsPresent = (
+        activityPassages &&
+        activityPassages[0] &&
+        !!activityPassages[0].image_link &&
+        !!activityPassages[0].image_alt_text &&
+        !!activityPassages[0].image_caption &&
+        !!activityPassages[0].image_attribution
+      );
+      return getCheckIcon(imageDetailsPresent);
+    case MAX_ATTEMPTS_FEEDBACK:
+      const maxAttemptsFeedbackPresent = (
+        activityBecausePrompt &&
+        activityButPrompt &&
+        activitySoPrompt &&
+        !!activityBecausePrompt.max_attempts_feedback &&
+        !!activityButPrompt.max_attempts_feedback &&
+        !!activitySoPrompt.max_attempts_feedback &&
+        activityBecausePrompt.max_attempts_feedback !== BREAK_TAG &&
+        activityButPrompt.max_attempts_feedback !== BREAK_TAG &&
+        activitySoPrompt.max_attempts_feedback !== BREAK_TAG
```

Comment:
Probably not worth bothering with here, but when comparing the same thing multiple times, I like to use `includes` over `&&`.  This could be:
```
![activityBecausePrompt.max_attempts_feedback,
 activityButPrompt.max_attempts_feedback,
 activitySoPrompt.max_attempts_feedback
].includes(BREAK_TAG)
```